src/main/native/cpp/MjpegServerImpl.cpp

```diff
@@ -9,23 +9,21 @@
 
 #include <chrono>
 
-#include "llvm/SmallString.h"
-#include "support/raw_socket_istream.h"
-#include "support/raw_socket_ostream.h"
-#include "tcpsockets/TCPAcceptor.h"
-
-#include "c_util.h"
-#include "cscore_cpp.h"
 #include "Handle.h"
 #include "HttpUtil.h"
 #include "JpegUtil.h"
 #include "Log.h"
 #include "Notifier.h"
 #include "SourceImpl.h"
+#include "c_util.h"
+#include "cscore_cpp.h"
+#include "llvm/SmallString.h"
+#include "support/raw_socket_istream.h"
+#include "support/raw_socket_ostream.h"
+#include "tcpsockets/TCPAcceptor.h"
```

Comment:
I really don't like this.  There's a newline in there for a reason.  It's separating the wpiutil includes from the includes local to this project.